test-cases/batch-tests/batch-tests.sh

```diff
@@ -31,19 +31,19 @@ latexindent.pl -l documentation.yaml -rv -w -m -s -g not-working.log doc*.tex
 
 # local settings found with default name (friend #1)
 latexindent.pl -s -l local-settings-friend-1/mwe.tex
-(cd local-settings-friend-1 && (cat indent.log | grep "defaultIndent" | sed 's/^ *//g' > localSettings.yaml))
+(cd local-settings-friend-1 && (cat indent.log | grep "WARN" > warnings.txt))
```

Comment:
Great, thanks! Is it possible to do this without changing directory?

Reply:
yes, but it is a shortcut for not having to type the directory twice; the longer (equivalent) of this particular line would read:

```diff
- (cd local-settings-friend-1 && (cat indent.log | grep "WARN" > warnings.txt))
+ cat local-settings-friend-1/indent.log | grep "WARN" > local-settings-friend-1/warnings.txt
```

as the command inside parenthesis `( ... )` is run in a subshell, the directory is never directly changed in this shell itself; if you want me to change it anyway, let me know.